### ISSUE TITLE
daily installation failing on WSL - unbuntu

### ISSUE THREAD

**Issue by MaryProSocial** [daily-co/daily-python#9]
Hello, I'm trying to use the daily-python package and I set up WSL (linux on Windows) with unbuntu.  pip install daily-python seems to work as expected, however, there is no daily folder in the build at lib/python3.10/site-packages.  Edit: I don't understand how the abi3.so file is working, but it looks like that is supposed to be able to be accessed in my code. Unfortunately it's not working for me in vscode - none of the modules are found in my code editor or during debug.

**Comment by aconchillo** [comment]
Hi @MaryProSocial. Thank you for reporting this issue. `daily-python` currently doesn't support Windows but WSL was actually something that we wanted to try. I'll try to give it a shot soon and report back.

An `abi3.so` is just a Python binary extension. `daily-python` is actually built from Rust code so, funny enough, there's actually no Python code involved. Currently, we don't provide all the source code so people can build it, since it depends on some internal libraries we haven't made public yet.

**Comment by MaryProSocial** [comment]
Hello again, I did finally get this working in WSL, although I'm not exactly sure how I did.  there were a couple of fidgety things - one my python instance wasn't pointed to .venv and I had to manually locate the path, which was a bit different in linux.  Second, I had installed some prerequisites for rust and c.  I don't know which of those issues was causing my problem, or if both were an issue, but thankfully the package is now working as expected.
